tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 isolated_build = true
-envlist = pylint,mypy,py{36,37,38},coverage-report
+envlist = pylint,mypy,py{36,37,38},coverage-report,mutmut
```

Comment:
Not sure if it is the right place to add a command, I am concerning regarding running `tox -p all -o` - it will include the `mutmut` build as well, it could be manually excluded, but I think it shouldn't be run by default at all. What do you think?